packages/xsnap/src/xsnap.c

```diff
@@ -3,6 +3,8 @@
 #include "xsSnapshot.h"
 #include "xs.h"
 
+#define XSNAP_VERSION "0.1.0"
```

Comment:
how about a test that this matches the package version?

Reply:
Ah, that's where it's supposed to come from?  I'll `-DXSNAP_VERSION=...` in the Makefiles.